apps/src/lab2/views/components/PanelContainer.tsx

```diff
@@ -50,43 +52,47 @@ const PanelContainer: React.FunctionComponent<PanelContainerProps> = ({
         <div
           className={classNames(
             'panelContainerHeader',
-            moduleStyles['panelContainerHeader-' + theme],
+            moduleStyles.panelContainerHeader,
+            moduleStyles[`panelContainerHeader${capitalizedTheme}`],
```

Comment:
minor nit on CSS style names: if we always expect to have a theme (e.g. never just `panelContainerHeader` by itself), it might be nice to extract the common styles into a placeholder selector (`%panel-container`) and then create light/dark variants that extend from it. Then that way, you just need to apply one style here (the light or dark variant) rather than the parent style plus the theme specific style.

Plus, if the names are lower case hyphenated, then we get away not needing to capitalize the first letter too 🙂 

Also, since we now can see readable class names on dev, should we remove the string `'panelContainerHeader'` classname?